pkg/util/winutil/process.go

```diff
@@ -0,0 +1,276 @@
+// +build windows
+
+package winutil
+
+import (
+	"fmt"
+	"unsafe"
+
+	"golang.org/x/sys/windows"
+)
+
+var (
+	modntdll                      = windows.NewLazyDLL("ntdll.dll")
+	modkernel                     = windows.NewLazyDLL("kernel32.dll")
+	procNtQueryInformationProcess = modntdll.NewProc("NtQueryInformationProcess")
+	procReadProcessMemory         = modkernel.NewProc("ReadProcessMemory")
+	procIsWow64Process            = modkernel.NewProc("IsWow64Process")
+)
+
+// C definition from winternl.h
+
+//typedef enum _PROCESSINFOCLASS {
+//    ProcessBasicInformation = 0,
+//    ProcessDebugPort = 7,
+//    ProcessWow64Information = 26,
+//    ProcessImageFileName = 27,
+//    ProcessBreakOnTermination = 29
+//} PROCESSINFOCLASS;
+
+type PROCESSINFOCLASS uint32
+
+const (
+	ProcessBasicInformation   = PROCESSINFOCLASS(0)
+	ProcessDebugPort          = PROCESSINFOCLASS(7)
+	ProcessWow64Information   = PROCESSINFOCLASS(26)
+	ProcessImageFileName      = PROCESSINFOCLASS(27)
+	ProcessBreakOnTermination = PROCESSINFOCLASS(29)
+)
+
+// IsWow64Process determines if the specified process is running under WOW64
+// that is, if it's a 32 bit process running on 64 bit winodws
+func IsWow64Process(h windows.Handle) (is32bit bool, err error) {
+	is32bit = false
```

Comment:
In golang the named returns would already had default values, so these initializations are not necessary.